src/main/docker/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-FROM opensuse:latest
+FROM opensuse/leap:latest
```

Comment:
Should we use the 15.0 or 15 tag instead of latest?

Reply:
No latest is fine - the first thing we do is update the OS packages anyway.